go/cmd/api-gateway/config_prod.go

```diff
@@ -0,0 +1,14 @@
+//go:build !local
+// +build !local
+
+package main
+
+import "go.uber.org/zap"
+
+var serviceName = "orchestrator"
+var port = ":8080"
```

Comment:
Should the port not be 8080 because of the orchestrator?

Reply:
Actually now that i think about it, they are on different pods so the same port should work fine